agent/mono-recorder-agent.c

```diff
@@ -0,0 +1,190 @@
+/**
+ *  Copyright 2019 SK Telecom Co., Ltd.
+ *    Author: Walter Lozano <walter.lozano@collabora.com>
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "config.h"
+
+#include "mono-recorder-agent.h"
+#include <hwangsae/recorder.h>
+#include <glib/gstdio.h>
+#include <gst/gst.h>
+
+#include <glib-unix.h>
+
+struct _HwangsaeMonoRecorderAgent
```

Comment:
May I suggest `HwangsaeSimpleRecorderAgent`? Mono makes a wrong impression it has something to deal with sound.

(I'll continue the review on Monday. Have a nice weekend.)

Reply:
Sure, not problem at all, `HwangsaeSimpleRecorderAgent` sounds nice.